sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -38,8 +53,79 @@ static Statsbeat()
 
         private static Measurement<int> GetAttachStatsBeat()
```

Comment:
Consider adding logging to this class.
Especially around the AMS. Just something informational that indicates that the request was made and SDK did/didn't receive a response.